compiler/src/dmd/expression.d

```diff
@@ -3790,6 +3795,7 @@ extern (C++) final class ArrayExp : UnaExp
 
     size_t currentDimension;    // for opDollar
     VarDeclaration lengthVar;
+    bool modifiable = false;    // is this expected to be an lvalue in an AssignExp? propagate to IndexExp
```

Comment:
I think this is missing from C++ headers.

Reply:
Good catch, added in expression.h. frontend.h was already updated.